src/Factory/Api/Mastodon/Status.php

```diff
@@ -288,10 +288,7 @@ public function createFromUriId(int $uriId, int $uid = 0, bool $display_quote =
 			$in_reply = [];
 		}
 
-		$queue_count   = $item['delivery_queue_count'];
-		$queue_done    = $item['delivery_queue_done'];
-		$queue_failed  = $item['delivery_queue_failed'];
-		$delivery_data = new FriendicaDeliveryData($queue_count, $queue_done, $queue_failed);
+		$delivery_data = new FriendicaDeliveryData($item['delivery_queue_count'] ?? 0, $item['delivery_queue_done'] ?? 0, $item['delivery_queue_failed'] ?? 0);
```

Comment:
Sorry for this forth and back. I just realized, that a value of `0` could be misleading. I guess we should transmit a `null` on each post that does not have got this information.

Reply:
As far as I can tell these aren't nullable columns in the database so this should be moot. Please confirm.

Reply:
The values are part of view. And in the view the join is done via some "left join", because we only have got this information on posts that left our system.

Reply:
Should allow null and not overrides to zero